.jazzy.yml

```diff
@@ -0,0 +1,174 @@
+author: Matt Broadstone, Kaitlin Mahar, and Patrick Freed
+author_url: https://github.com/mongodb/mongo-swift-driver
+github_url: https://github.com/mongodb/mongo-swift-driver
+theme: fullwidth
+swift_build_tool: spm
+readme: "etc/docs-main.md"
+documentation: "Guides/*.md"
+
+custom_categories:
+  - name: General Guides
+    children:
+      - Development
+      - Multithreaded-Usage
+      - BSON-Guide
+
+  - name: Core Types
+    children:
+      - MongoClient
+      - MongoClientOptions
+      - MongoDatabase
+      - MongoDatabaseOptions
+      - MongoCollection
+      - MongoCollectionOptions
+
+  - name: BSON Types
```

Comment:
I went ahead and included the BSON symbols here since it seems like it will be possible for us to keep including them in the driver docs even when they're coming from the separate repo.

I figured I would wait to deal with updating the script to try to do that cross-repo combination until we've deleted the BSON code in the driver and switched to depending on the new library so I opened [SWIFT-1007](https://jira.mongodb.org/browse/SWIFT-1007).